### DISCUSSION TITLE
Error logging in from Android app, empty client and server logs

### DISCUSSION THREAD

**Discussion by mpeter50** [immich-app/immich#5545] [Q&A]
Hi!

I was trying to log in in the Immich android app to my server, but I get an error popup I can't do anything with, and no logs about the issue either in the android logcat or the server logs.

My server is on v1.90.0, and it is just a minimal test install, so it would go through plain HTTP on the local network. I can log in and it seems to generally work when logging in to it from my PC with a web browser.
The app is on v1.72.1, as that is the last version available for my phone.

The versions are quite different I see, but I don't think this is about the version mismatch, as even the server does not tell a line that it received a request it couldn't do anything with, and the error popup is more about a connection error. But as neither of the software has logged anything since server startup (except redis every 5 minutes), I don't know what is the problem.

**Comment by shenlong-tanwen** [comment]
The issue is definitely because of the version mismatch. There were a lot of backward incompatible changes between your current mobile app version and your server version. Try updating your mobile app and check if it works.

**Reply by mpeter50** [reply]
I see. I cannot update the app since you have increased the minimum supported version for google play's policy (as I understand).
However I'm installing the app from F-droid, and the minimum supported Android version was decreased there too.

In fact, I have tried to build the app myself with a lower minsdk setting so that it installs on my phone.
The build has completed and the app has started.. but it stays on the splash screen, with the Immich logo and no text.
The app [doesn't log](https://gist.github.com/mpeter50/3aa6ea1abffbc7b41bba661dcd5385d6) anything actionable this time either, no error or warning level messages at all, just that dart has started some server on localhost.

Why are there no log messages in the app, and the server too? Do I need to turn them on? If so, how do I do that?
Honestly, even after reading lots of positive feedback about Immich, this does not seem to be very promising, because if some problem occurs and Immich stops working I won't be able to fix it if it doesn't tell me or even give signs on whats the problem.

**Comment by shenlong-tanwen** [comment]
> Why are there no log messages in the app, and the server too? Do I need to turn them on? If so, how do I do that?

You don't have to turn on anything for the logging to show up. You did not have any error in the server log because there is no error in the server side to be logged. The issue with the backward incompatible changes are usually with how the android app handles the response from the server. The logging in the mobile app might not be the greatest but we are improving it steadily. Can you please share the details of the device in which you're testing the app on, along with the android sdk version?
